--- util/superiotool/superiotool.h	2018-05-16 21:00:17.000000000 +0200
+++ util/superiotool/superiotool.h	2018-08-27 12:50:25.627387315 +0200
@@ -25,7 +25,7 @@
 #include <stdint.h>
 #include <string.h>
 #include <getopt.h>
-#if defined(__GLIBC__)
+#if  !(defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__))
 #include <sys/io.h>
 #endif
 #if (defined(__MACH__) && defined(__APPLE__))
